tests/postgres_tests/test_bulk_update.py

```diff
@@ -12,7 +14,8 @@
     pass  # psycopg2 isn't installed.
 
 
-class BulkSaveTests(PostgreSQLTestCase):
+@modify_settings(INSTALLED_APPS={'append': 'django.contrib.postgres'})
+class BulkSaveTests(PostgreSQLHStoreTestCase):
```

Comment:
You've added `PostgreSQLHStoreTestCase` but left the `@modify_settings` decorator in place.

I think Adam only [suggested](https://github.com/django/django/pull/12646/files#r453747532) seeing if the decorator could be applied to `PostgreSQLTestCase` in general. Given that it didn't seem to work I'd just stick with the decorator here and not bother with creating a separate class.

(Also `PostgreSQLHStoreTestCase` isn't a great name for is as it is more like a `PostgreSQLWithInstalledContribTestCase`... But let's not do that! 😆)